Documentation/building/cross-building.md

```diff
@@ -2,7 +2,7 @@ Cross Compilation for ARM on Linux
 ==================================
 
 Through cross compilation, on Linux it is possible to build CoreFX for arm, arm-softfp or arm64.
```

Comment:
nit: awkward sentence.  It feels odd to start this document with "Though cross compilation..."  I had to read the sentence twice.  Perhaps something more like...

"It is possible to build CoreFx on Linux for arm, arm-softfp, or arm64 by cross compiling."